config/storage.yml

```diff
@@ -4,7 +4,11 @@ test:
 
 local:
   service: Disk
-  root: <%= Rails.root.join("storage") %>
+  root: <%= Rails.root.join(Settings.file_uploads_root, 'draft') %>
```

Comment:
Yes, this does the right thing when given relative (local/dev envs) and absolute (deployed/prod envs) paths.

Reply:
Should we just add `/draft` into the `file_uploads_root` variable?

Reply:
No, because the draft subdir isn't the root. Maybe I'm not grokking, though.

Reply:
Ah, I see.